### DISCUSSION TITLE
Is there a way to automatically refactor all Doctrine ORM Annotations to PHP 8 attributes?

### DISCUSSION THREAD

**Discussion by ruudk** [rectorphp/rector#6199] [Q&A]
It's not automatically done when using the `PHP_80` set so maybe it's not there yet?

**Comment by TomasVotruba** [comment]
Hi, where are the Doctrine attributes released?

**Reply by ruudk** [reply]
Hmm, I see it will be released in 2.9.x https://github.com/doctrine/orm/tree/2.9.x

**Reply by TomasVotruba** [reply]
Ah, it's on the "promise" list last year :D

Could you link the PR?

The `DOCTRINE_ORM_290` set should cotain it then: https://github.com/rectorphp/rector-doctrine

This is how `SYMFONY_52` is migrating annotations to attributes:
https://github.com/rectorphp/rector-symfony/blob/eebadd7276647691630695513983f71d6ce893a0/config/sets/symfony/symfony52.php#L33-L42

**Reply by ruudk** [reply]
see https://github.com/doctrine/orm/pull/8266
